staging/src/github.com/openshift/sdn/cmd/sdn-cni-plugin/openshift-sdn_linux.go

```diff
@@ -194,6 +194,7 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 			return fmt.Errorf("failed to configure container loopback: %v", err)
 		}
 
+		var addrs []netlink.Addr
```

Comment:
hm... I think it would be clearer to move the definition of `dsts` out to here, and leave `addrs` and the loop over it inside the macvlan-specific code